satinfo/cris-fsr_n20/readme

```diff
@@ -1,2 +1,2 @@
 NOAA 20 (JPSS-1) launced 20171118
-turned on in operational satinfo 2018053012.  entries taken from global_satinfo.txt.2020022012
+turned on in operational satinfo 2018053012  
```

Comment:
is that a relevant comment, since the new file is stamped with 20180217

Reply:
it was used by NASA starting 20180217, but NCEP ops didn't use it till 2018053012